package.json

```diff
@@ -47,8 +48,10 @@
     "metismenujs": "^1.0.3",
     "popper.js": "^1.15.0",
     "punycode": "1.4.1",
-    "roboto-fontface": "^0.10.0",
     "qrcode-generator": "^1.4.3",
+    "roboto-fontface": "^0.10.0",
+    "sass": "^1.23.0-module.beta.1",
+    "sass-loader": "^8.0.0",
```

Comment:
Are you sure, that this new dependencies are needed?

Reply:
Line 52 - 54 is not needed now, that's right.